src/modules/GroupChannelList/context/GroupChannelListProvider.tsx

```diff
@@ -66,12 +65,12 @@ const initialState: GroupChannelListState = {
   allowProfileEdit: false,
   isTypingIndicatorEnabled: false,
   isMessageReceiptStatusEnabled: false,
-  onChannelSelect: noop,
-  onChannelCreated: noop,
-  onThemeChange: noop,
-  onCreateChannelClick: noop,
-  onBeforeCreateChannel: null,
-  onUserProfileUpdated: noop,
+  onChannelSelect: undefined,
+  onChannelCreated: undefined,
+  onThemeChange: undefined,
+  onCreateChannelClick: undefined,
+  onBeforeCreateChannel: undefined,
+  onUserProfileUpdated: undefined,
```

Comment:
We don't handle the undefined functions on the other files, like optional chaining.

We should confirm if there's no Function undefined error, during run-time.
or use `noop` again

Reply:
This is updated like the comment below by Irene.